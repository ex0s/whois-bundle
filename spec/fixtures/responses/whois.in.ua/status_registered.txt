% In.UA whois server. (whois.in.ua)
% All questions regarding this service please send to help@whois.in.ua
% To search for domains and In.UA maintainers using the web, visit http://whois.in.ua
domain:      dle.in.ua
descr:       dle.in.ua
admin-c:     VP535-UANIC
tech-c:      NIC-UANIC
status:      OK-UNTIL 20131218000000
nserver:     NS12.UADNS.COM 
nserver:     NS11.UADNS.COM 
nserver:     NS10.UADNS.COM 
mnt-by:      DRS-MNT-INUA
mnt-lower:   DRS-MNT-INUA
changed:     notify@drs.ua 20121216134104
source:      INUA

