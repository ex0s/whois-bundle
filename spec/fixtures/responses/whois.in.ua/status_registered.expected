#status
  %s == :registered

#available?
  %s == false

#registered?
  %s == true


#created_on
  %s %ERROR{AttributeNotSupported}

#updated_on
  %s %CLASS{time}
  %s %TIME{2012-12-16 13:41:04}

#expires_on
  %s %CLASS{time}
  %s %TIME{2013-12-18 00:00:00}


#nameservers
  %s %CLASS{array}
  %s %SIZE{3}
  %s[0] %CLASS{nameserver}
  %s[0].name == "ns12.uadns.com"
  %s[1] %CLASS{nameserver}
  %s[1].name == "ns11.uadns.com"
  %s[2] %CLASS{nameserver}
  %s[2].name == "ns10.uadns.com"
